lib/resources/r4/related_person.yaml

```diff
@@ -75,4 +76,313 @@ fields:
+            }
+          }
+      ]
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/related-person-encounter?_format=json
+
+- name: Relationship Level
+  required: 'No'
+  type: Extension
+  description: The resource's relationship to either the patient or encounter level.
+  action: create
+  note: |
+    <ul>
+      <li>If provided, the <code>valueCodeableConcept.coding[0].code</code> must be `Encounter`.</li>
+    </ul>
+  example: |
+    {
+      "extension": [
+          {
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/relationship-level",
+                "valueCodeableConcept": {
+                    "coding": [
+                      {
+                          "system": "http://hl7.org/fhir/resource-types",
+                          "code": "Encounter",
+                      }
+                    ]
+                }
+          }
+      ]
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/relationship-level?_format=json
+
+- name: identifier
+  required: 'No'
+  type: List of Identifier
+  description: The identifiers for the related person.
+  action: create
+  example: |
+    {
+      "identifier": [
+        {
+          "type": {
+            "coding": [
+              {
+                "code": "MR",
+                "system": "http://hl7.org/fhir/v2/0203"
+              }
+            ]
+          },
+          "system": "urn:oid:1.1.1.1.1.1",
+          "value": "&lt;unique identifier&gt;",
+          "period": {
+            "start": "2016-01-02T00:00:00-05:00",
+            "end": "2020-01-02T00:00:00-05:00"
+          }
+        }
+      ]
+    }
+  note: |
+    <ul>
+      <li> Subsequent identifiers may be provided, but have the following constraints:
+        <ul>
+          <li> Must not contain the `use` fields. </li>
+          <li> Must contain the `type`, `system`, and `value` fields. </li>
+          <li> When specifying a `period`, must include a time component with a timezone. </li>
+        </ul>
+      </li>
+    </ul>
+  url: http://hl7.org/fhir/R4/datatypes.html#Identifier
+
+- name: address
+  required: 'No'
+  type: List of Address
+  description: Addresses for the individual.
+  action: create
+  example: |
+    {
+      "address": [
+        {
+          "use": "home",
+          "line": [
+             "1212 Faircastle Drive",
+             "Apartment 406"
+          ],
+          "city": "KC",
+          "district": "Jackson",
+          "state": "KS",
+          "postalCode": "64199",
+          "country": "United States of America",
+          "period": {
+            "start": "2012-05-17T15:33:18.000Z"
+          }
+        }
+      ]
+    }
+  note: |
+    <ul>
+      <li> Each address must contain the `use` field. </li>
+      <li> Each address must not contain the `text` field. </li>
+      <li> When specifying a `period`, the fields must include a time component with a timezone. </li>
+    </ul>
```

Comment:
Should we mention only the first 4 address lines are written?

Reply:
Added here: https://github.com/cerner/fhir.cerner.com/pull/527/commits/1f36d5e5aed0443c83faefe32f66d2417a42a276#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R284